### ISSUE TITLE
Acomodar nome no Billing Adress e Shipping Address no sistema

### ISSUE THREAD

**Issue by JulianReina96** [Murphyly/mate85_ecommerce#94]
Acho que precisa na tela de StatusPedido, também na tela de admin de pedido, mas pode ter em mais locais
Ou seja, mostrar junto com o endereço o nome da pessoa para qual está sendo enviado o pedido

**Comment by JulianReina96** [comment]
![Image](https://github.com/Murphyly/mate85_ecommerce/assets/87534209/0fcd9da3-09a1-41c6-a7e3-9ae82a499587)

A tela de Status pedido já possui o nome junto ao endereço. Essa issue é sobre isso?

@Shirkit @Murphyly

**Comment by Shirkit** [comment]
Sim, a issue é sobre isso. Esse é um dos locais. Outros locais envolvem a tela de listar pedido no admin, eu coloquei aqui para ver se teria outros.
